.storybook/main.js

```diff
@@ -49,6 +49,7 @@ module.exports = /** @type {Omit<StorybookConfig,'typescript'|'babel'>} */ ({
             loose: true,
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26757&commitId=a30099462e893449c68e30b4f2814b8c668b3ee9&env=prod)

✅ There was 0 screenshots added, 1 screenshots removed, 1842 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+inactive+badge.normal.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv9"/>